projects/power_distribution/src/gpio.c

```diff
@@ -1,5 +1,5 @@
-#include "power_distribution_gpio.h"
-#include "power_distribution_events.h"
+#include "gpio.h"
+#include "events.h"
```

Comment:
I would like to keep some form of a prefix on `events` since just `events.h` sounds like an ms-common library. `pd_events.h` would follow the naming convention of e.g. `mci_events.h`, `solar_events.h` in other projects.